sskittenapi/catapi/catapi/config.py

```diff
@@ -33,6 +33,15 @@ def _get_comma_separated_env_variable(name: str) -> List[str]:
 ELASTIC_APM_ENABLED = _get_boolean_env_variable("ELASTIC_APM_ENABLED")
 ELASTIC_APM_SERVER_URL = os.getenv("ELASTIC_APM_SERVER_URL")
 
+ALLOWED_ORIGINS_REGEXPES = [
+    "http://localhost[^\\.]*$",
+    "https://ai\\.gengo\\.test$",
+    "https://.*\\.telusinternational\\.ai$",
+    "https://.*\\.lionbridgeai\\.com$",
+    "https://.*\\.lionbridge\\.com$",
+    "capacitor://localhost$",
```

Comment:
[suggestion] It is better to add domains when needed than to have all of them for "just-in-case" and end up not using any of those..
```suggestion
```
WDYT?